[CMake problem] LINK : fatal error LNK1104: unable to open file 'bfg::lyra.lib'

I sticked to the instructions written in your `CMakeLists.txt` which are:
* in your host project, put `add_subdirectory( <path-to-lyra> )` - checked `add_subdirectory(vendor/Lyra-master ./liblyra)`
* link to bfg::lyra with `target_link_libraries( <my-exe/lib> PUBLIC bfg::lyra )` - checked `target_link_libraries(ao-avalonian-roads-gps LINK_PUBLIC bfg::lyra)`
 
On windows, I get the late link error
> LINK : fatal error LNK1104: unable to open file 'bfg::lyra.lib'

Sure, the directory where the lib should be is empty (in my case build\liblyra\Debug).

Is Lyra supposed to build just by adding the add_subdirectory? Because it has its own .sln... which could be a problem I guess.

More info incomming from Linux:
> Target "ao-avalonian-roads-gps" links to target "bfg::lyra" but the target was not found.  Perhaps a find_package() call is missing for an IMPORTED target, or an ALIAS target is missing?

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -39,8 +39,7 @@ target_include_directories(
   )
 
 # Add an alias to public name.
-add_library( bfg::Lyra ALIAS lyra )
-
+add_library( bfg::lyra ALIAS lyra )
 
 ## Installation Code
 include(GNUInstallDirs)

